worker/backend/spec/mounts.go

```diff
@@ -0,0 +1,59 @@
+package spec
+
+import "github.com/opencontainers/runtime-spec/specs-go"
+
+var (
+	InitMount = specs.Mount{
+		Source:      "/usr/local/concourse/bin/init",
+		Destination: "/tmp/gdn-init",
+		Type:        "bind",
+		Options:     []string{"bind"},
+	}
+
+	AnyContainerMounts = []specs.Mount{
+		InitMount, // ours
+
```

Comment:
same question. Doesn't containerd have some sort of default mounts for containers?

Reply:
yeahh, it does have some defaults - when you do `containerd.WithNewSpec()`, under the hood, it comes down to https://github.com/containerd/containerd/blob/4cd5de74bd0ba0179a9fec15135c0d5170df5070/oci/spec.go#L132, which contains the defaults you mean (I think 😅 )

As you can see in that file, that's all coming from a set of private methods that we don't really have control over.

I preferred taking the approach of going with Garden's defaults (which, btw, are also private), as we can then have total control over them.

Wdyt?

---


As I mentioned above, it might be interesting to diff those two though 🤔